rusk/src/bin/args.rs

```diff
@@ -54,6 +54,10 @@ pub struct Args {
     /// path to encrypted BLS keys
     pub consensus_keys_path: Option<PathBuf>,
 
+    #[clap(long)]
+    /// height at which migration will be performed
+    pub migration_height: Option<u64>,
```

Comment:
Is it right that we want to put this in as a command line argument? It seems to me this should either be provided by the consensus by some mechanism, or baked-in/hard-coded into the binary.

Reply:
We're going to provided it as configuration in the toml file.
It will be rollout by the itn-installer.

Even if I agree with your proposal about consensus mechanism, that's not part of the current ITN scope